### ISSUE TITLE
The WHO is reputable

### ISSUE THREAD

**Issue by rohancalum** [larrysalibra/WearAFuckingMask.com#6]
"But the Fucking WHO and some Fucking governments told me not to wear a Fucking mask.

Fuck those Fucking Fuckers, they Fucking let this Fucking virus spread out of control by being more worried about their Fucking political careers than saving Fucking lives."

Needs to go. Trying to subvert the government/WHO advisories is incredibly irresponsible. Your argument still stands without this section.

**Comment by ghost** [comment]
he is not trying to subvert anyone, nothing wrong with saying the truth ... oh well I see ...

**Comment by shayneoneill** [comment]
Yeah please remove this part. Its actually really important people listen to the WHOs advice on this. They ARE the responsible authority, and its irresponsible at a time when disinfo is running rampant to suggest otherwise. Yes they may have goofed up the early stages. Humans are humans. But that does not mean theres woowoo or incompetence coming out of them.

**Comment by adrianmcli** [comment]
I agree that the WHO is a reputable organization but it's also worth noting that the WHO is _not_ saying "do not wear a mask". Therefore, I think rephrasing towards a less aggressive stance is probably a good idea.

That being said, read on for my analysis (and opinion) on what the WHO is saying.

**A very important point:** The WHO (and other institutions) are providing a nuanced message that is phrased in a way to take into consideration mask shortages of varying levels in different countries.

## What the WHO actually says

Let us take a look at what they officially say here: 
https://www.who.int/emergencies/diseases/novel-coronavirus-2019/advice-for-public/when-and-how-to-use-masks

> - If you are healthy, you only need to wear a mask if you are taking care of a person with suspected 2019-nCoV infection.

Another way to re-phrase this is "you do not need to wear a mask if (1) you are healthy and (2) you are not taking care of someone with a suspected infection".  However, given that we know non-symptomatic transmission is possible, there is actually no way of knowing whether or not (1) or (2) is true or not without testing.

> - Wear a mask if you are coughing or sneezing.

You also never know when you (or someone else) can start coughing or sneezing. It's not like I can predict whether or not I will sneeze in the morning before I head out for work.

> - Masks are effective only when used in combination with frequent hand-cleaning with alcohol-based hand rub or soap and water.

Lack of hand washing neutralizes a lot of the good that wearing a mask can help with. No controversy there.

> - If you wear a mask, then you must know how to use it and dispose of it properly.

This is important and I think everyone should evaluate the instructions provided.

--- 

Implicit in the first quoted statement above is that having a mask can indeed protect you (however imperfectly) from contracting the virus. When keeping in mind that certain countries are running out of masks, we can see why the WHO has adopted the phrasing here.

One alternative to point out is that homemade masks or re-usable fabric masks can also help (however imperfectly). The key point I want to highlight is that you do not know whether you or those around you are infected and therefore _**it is prudent to wear a mask provided that you are not contributing to any undue shortages in your locale**_.

All that being said, I do think the original language is too strong and requires a more nuanced explanation.

**Comment by soumasandesu** [comment]
Agreed to the evaluation by @adrianmcli .

> You also never know when you (or someone else) can start coughing or sneezing. It's not like I can predict whether or not I will sneeze in the morning before I head out for work.

> The key point I want to highlight is that you do not know whether you or those around you are infected and therefore it is prudent to wear a mask provided that you are not contributing to any undue shortages in your locale.

We also never know a person might be 
- infected by 2019 n-CoV, or
- are being ordered to stay at home for isolation

on street by just looking at his/her attitude/behavior. 

Some cases of infection are also shown that it's possible not having any symptoms if a person is mildly infected. 

You, as a responsible person, may prevent yourself from spreading cold/flu/virus to others by wearing a mask, is not meant to be that another person is as same responsible as you are. 

There may be even some crazy guys intended to spit/sneeze on street or other people/shops (esp they don't like), and such things have already happened before and reported on newspapers.

It's also why many people in HK & TW are wearing masks (or even respirators) and wash hands very frequently at all time on outside of their homes (esp crowded areas) from very early days of outbreak in Wuhan, China.

---

> Implicit in the first quoted statement above is that having a mask can indeed protect you (however imperfectly) from contracting the virus. When keeping in mind that certain countries are running out of masks, we can see why the WHO has adopted the phrasing here.

Actually, TW had also telling to their nationals not to use overuse masks when not visiting crowded area (news reported this in ~ mid Feb?), e.g. country park. [Some MDs](https://www.facebook.com/moka700720/posts/512857549428813) are also evaluated that the TW govt had such appeal because of shortage of surgical mask a.t.m. in TW, and supporting it. Recently, as their govt and the enterprises are hurrying up to build surgical mask machines + factories, the steadiness of mask supplying to the general public, and the seriousness of outbreak around the world, some other MDs are also said that ["it's not applicable anymore, and better to wear masks at all time"](https://today.line.me/tw/pc/article/%E5%81%A5%E5%BA%B7%E8%80%85%E4%B8%8D%E6%88%B4%E5%8F%A3%E7%BD%A9%EF%BC%81%E3%80%80%E6%84%9F%E6%9F%93%E7%A7%91%E4%B8%BB%E4%BB%BB%EF%BC%9A%E7%8F%BE%E5%9C%A8%E4%B8%8D%E9%81%A9%E7%94%A8-yvLxRo).

---

The original author might be using a bit too aggressive tone to express their _feeling_ that WHO and some govt are too _late_ to respond and telling info somewhat too _weak_ to citizens, regardless of the tone of whole context. This is somewhat hardly acceptable to everyone in different state/city/sovereignty where their infection rate/routes of spreading/health care system loading/... are different. For example, such aggressive tone of speech would be more accepted by people living in far eastern places e.g. HK, TW, JP, KR.

But I'd also like to notice that it's also important that everyone should have deep evaluation, for how much personal protection is **actually** enough for themselves **sensibly**, whilst not contributing to any undue shortages in your locale, as you @adrianmcli said.

**Comment by shayneoneill** [comment]
We're computer guys second guessing actual epidemiologists. Come on guys, come at this from a third person perspective. Are we engaging in woo when we , non experts, try and pick holes in experts because it doesnt fit our gut feeling or intuition.

This is the kind of thinking that powers antivaxxers, climate change denialists and the folks who self diagnose with everything under the DSM. 

Take a step back on this. IF the experts are saying something different to what we're saying, then its arrogance to believe the experts are wrong on this.

**Comment by adrianmcli** [comment]
The experts ARE saying to wear masks if there is a possibility that either you or those around you are infected. The converse is actually what is woo and trying to rationalize gut feelings. I've tried to avoid wearing masks for the longest time until I took a look at the literature and what the authorities are saying. Please point me to actual text from the WHO that tells people not to wear face masks.

**Comment by hkgsherlock** [comment]
> Take a step back on this. IF the experts are saying something different to what we're saying, then its arrogance to believe the experts are wrong on this.

**Comment by rohancalum** [comment]
My issue pertained to this site literally denouncing the word of the WHO by saying "fuck those fuckers". My ask is to remove the instances where this is the case. The site should ENDORSE and SHARE what the WHO is saying... one could say, "Listen to the Fucking WHO" and I'd be happy.

**Comment by adrianmcli** [comment]
@rohancalum I agree 100%

Would you like to put in a PR?

**Comment by sctse999** [comment]
"There is no specific evidence to suggest that the wearing of masks by the mass population has any potential benefit. In fact, there's some evidence to suggest the opposite in the misuse of wearing a mask properly or fitting it properly," Dr. Mike Ryan, executive director of the WHO health emergencies program

https://edition.cnn.com/2020/03/30/world/coronavirus-who-masks-recommendation-trnd/index.html

**Comment by adrianmcli** [comment]
The CDC has now reversed course (or more graciously, "adjusted" their messaging) to now recommend the general population wear homemade cloth masks: https://www.cdc.gov/coronavirus/2019-ncov/prevent-getting-sick/cloth-face-cover.html

**Comment by sctse999** [comment]
After over 1 million people infected, WHO director said that wearing a mask isn't a bad idea. 

https://www.bloomberg.com/news/videos/2020-04-03/who-s-ryan-says-using-homemade-face-masks-isn-t-a-bad-idea-video

The original language in this website doesn't seem strong to me. Not at all.

**Comment by sctse999** [comment]
Advice on the use of masks in the context of COVID-19 on 6 Apr 2020, by WHO

>> As described above, the wide use of masks by healthy people in the community setting is not supported by current evidence and carries uncertainties and critical risks.

WHO is reputable? 

https://apps.who.int/iris/bitstream/handle/10665/331693/WHO-2019-nCov-IPC_Masks-2020.3-eng.pdf?sequence=1&isAllowed=y

**Comment by shayneoneill** [comment]
This is degenerating into pseudoscientific woo. Those documents are well supported by the reference. It would be irresponsible of me to support this project anymore. Best of luck.

**Comment by bvdlingen** [comment]
WHO on masks:

**Wearing  a  medical mask is   one  of  the  prevention  measures that can limit the spread of certain respiratory viral diseases, including COVID-19.** 
However, the use of a mask alone is insufficient to provide an adequate level of protection, and other  measures  should  also be  adopted. 
Whether  or  not masks are used, maximum compliance with hand hygiene and other IPC  measures is  critical  to  prevent  human-to -human transmission of COVID-19.

**Comment by adrianmcli** [comment]
That is a relatively recent change to their messaging. This was not the messaging from the WHO previously when most of the above conversations took place.
